release-schema.json

```
@@ -13,8 +13,54 @@
         },
         "legalBasis": {
           "title": "Legal basis",
-          "description": "The legal basis of the procedure.",
-          "$ref": "#/definitions/Classification"
+          "description": "The laws and regulations that govern the contracting process and grant legal authority to the procuring entity.",
```

There's only one reference, so making it singular. Applying style guide re: procuring entity.

```suggestion
          "description": "The law or regulation that governs the contracting process and grants legal authority to the buyer or procuring entity.",
```